pkg/monitoring/top/top_windows.go

```diff
@@ -6,25 +6,23 @@ import (
 	"log"
 	"time"
 
-	"github.com/cloudradar-monitoring/cagent/perfcounters"
+	"github.com/cloudradar-monitoring/cagent/pkg/monitoring"
+
 	"github.com/pkg/errors"
 )
 
-var watcher *perfcounters.WinPerfCountersWatcher
 var counterPath = "\\Process(*)\\% Processor Time"
 
 func (t *Top) startCollect(interval time.Duration) {
-	watcher = perfcounters.Watcher()
-
-	err := watcher.StartContinousQuery(counterPath, interval)
+	err := monitoring.GetWatcher().StartContinuousQuery(counterPath, interval)
 	if err != nil {
 		log.Printf("Failed to StartQuery: %s", err)
```

Comment:
Why not error?
Also, please can you prepend context, like `[VM]` because this is a global logging message. We did this way in the other parts of the code

Reply:
Oh, sorry. This line wasn't made by you. Maybe @soupdiver can address it here on in separate PRs
Still, you have other logging where you could prepend the context ("[VM]")

Reply:
done